api/proto/teleport/legacy/types/types.proto

```diff
@@ -2325,6 +2325,19 @@ message RoleSpecV6 {
   ];
 }
 
+// CreateHostUserMode determines whether host user creation should be
+// disabled or if host users should be cleaned up or remain after
+// sessions end.
+enum CreateHostUserMode {
+  HOST_USER_MODE_UNDEFINED = 0;
```

Comment:
nit: [Use "*_UNSPECIFIED"](https://protobuf.dev/programming-guides/dos-donts/#do-include-an-unspecified-value-in-an-enum)
```suggestion
  HOST_USER_MODE_UNSPECIFIED = 0;
```